README.md

```diff
@@ -6,20 +6,12 @@
 This package provides the [Simple Icons 15.7.0](https://github.com/simple-icons/simple-icons/releases/tag/15.7.0)
 packaged as a set of [React](https://facebook.github.io/react/) components.
 
-  <a href="https://www.npmjs.com/package/@icons-pack/react-simple-icons" target="_blank">
-    <img src="https://img.shields.io/npm/v/@icons-pack/react-simple-icons?color=CB061D&style=flat-square" alt="www.npmjs.com!" />
-  </a>
-
-  <a href="https://www.npmjs.com/package/@icons-pack/react-simple-icons" target="_blank">
-    <img src="https://img.shields.io/npm/dw/@icons-pack/react-simple-icons?color=087BB4&style=flat-square" alt="downloads" />
-  </a>
-
-  <a href="https://github.com/icons-pack/react-simple-icons/blob/canary/LICENSE" target="_blank">
-    <img src="https://img.shields.io/npm/l/@icons-pack/react-simple-icons?color=008660&style=flat-square" alt="licence" />
-  </a>
+  <a href="https://www.npmjs.com/package/@icons-pack/react-simple-icons" target="_blank"><img src="https://img.shields.io/npm/v/@icons-pack/react-simple-icons?color=CB061D&style=flat-square" alt="www.npmjs.com!" /></a>
+  <a href="https://www.npmjs.com/package/@icons-pack/react-simple-icons" target="_blank"><img src="https://img.shields.io/npm/dw/@icons-pack/react-simple-icons?color=087BB4&style=flat-square" alt="downloads" /></a>
+  <a href="https://github.com/icons-pack/react-simple-icons/blob/canary/LICENSE" target="_blank"><img src="https://img.shields.io/npm/l/@icons-pack/react-simple-icons?color=008660&style=flat-square" alt="licence" /></a>
```

Comment:
Let's respect and keep the author's code style. It looks like you just "fixed" a wrong link for the license. However, the license link you provided is incorrect. I don't think we have a `canary` branch.

Reply:
The code style is exactly the cause of the blue underline. [reference](https://github.com/orgs/community/discussions/79043#discussioncomment-7990622).

I did not change any links. Just removed the extra spacing between `<a>` and `<img />`. but yeah looks like the link in the current README is incorrect.

Reply:
Well github ignores target=“_blank” attributes. We can also use markdown style image embed instead of html. that might make it look better

Reply:
Can you explain why you submit the `canary` here? This doesn't seem like a human mistake.

Reply:
Ah, I see.